File: ambeth/Ambeth.Mapping/ambeth/mapping/IListTypeHelper.cs
using System;
using System.Collections;

namespace De.Osthus.Ambeth.Mapping
{
    public interface IListTypeHelper
    {
        bool IsListType(Type itemType);

        Object PackInListType(IEnumerable referencedVOs, Type listType);

        Object UnpackListType(Object item);
    }
}

File: jambeth/jambeth-mapping/src/main/java/com/koch/ambeth/mapping/IListTypeHelper.java
package com.koch.ambeth.mapping;

import java.util.Collection;

public interface IListTypeHelper
{
	<L> L packInListType(Collection<?> referencedVOs, Class<L> listType);

	Object unpackListType(Object item);

	boolean isListType(Class<?> type);
}